soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -738,8 +738,7 @@ void InitTrackerData(bool isDebug) {
         }
     }
     UpdateAllOrdering();
-    UpdateInventoryChecks();
-}
+} 
```

Comment:
```suggestion
}
```
cleaning up my previous build kick off